hnginternship5/kidstories-android
# FT-Implemented Authentication before user create story

### TITLE OF PR

User create story
### PURPOSE OF PR

User Create Story.
### FEATURES IMPLEMENTED

Add Story
### HOW TO TEST
When a user clicks the add story button the user should not be able to add story if not logged in
### TRELLO BOARD  LINK

https://trello.com/c/rNfmKHGR/2-user-create-stories-bemd
